juriscraper/opinions/united_states/state/coloctapp.py

```diff
@@ -50,3 +56,32 @@ def _process_html(self) -> None:
                     "status": "Published",
                 }
             )
+
+    def extract_from_text(self, scraped_text: str) -> Dict[str, Any]:
+        """Pass scraped text into function and return data as a dictionary
+        Notes for 'Citation':
+            - Reporter key for this court: 'COA'
+            - Type for a state: 2
+        :param scraped_text: Text of scraped content
+        :return: metadata
+        """
+        match_re = r"The.*SUMMARY.*?(?P<citation>\d{4}COA\d+).*?No\. (?P<docket>\d+CA\d+), (?P<headnotes>.*?)\n{2,}(?P<summary>.*?)COLORADO COURT OF APPEALS"
+        match = re.findall(match_re, scraped_text, re.M | re.S)
+        if not match:
+            metadata = {}
+        else:
```

Comment:
This is more stylistic but I think it would be nicer to do this 

```
metadata = {}
if match:
    volume, page = match[0][0].split("CO")
    metadata = {
                "OpinionCluster": {
                    "docket_number": match[0][1],
                },
                "Citation": {
                    "volume": volume.strip(),
                    "reporter": "CO",
                    "page": page.strip(),
                    "type": 2,
                },
       }
```

Reply:
This was for CO but its the if statement pattern that im getting at.

Reply:
Verification structure changed.